properties/P000149.md

```diff
@@ -1,15 +1,24 @@
 ---
 uid: P000149
-name: $\varepsilon$-space
+name: $\omega$-Lindelöf
 aliases:
-  - $\omega$-Lindelöf
+  - $\varepsilon$-space
 refs:
 - doi: 10.1016/0166-8641(82)90065-7
   name: Some properties of C(X), I (Gerlits & Nagy)
+- zb: '1153.54009'
+  name: Selection principles related to α-properties (Kočinac)
 ---
 
 Every $\omega$-cover of $X$ has a countable $\omega$-subcover.  (A family $\mathcal U$ of open subsets of $X$ is called an *$\omega$-cover* if each finite subset of $X$ is contained in an element of $\mathcal U$ and $X \notin \mathcal U$.)
 
 Equivalently, every finite power of $X$ is {P18}.
 
 See {{doi:10.1016/0166-8641(82)90065-7}}; the equivalence with every finite power being {P18} is on page 156.
+
+The name $\omega$-Lindelöf used in e.g.
+{{zb:1153.54009}} is motivated as the adaptation of
+{P18} for $\omegea$-covers. This property is
```

Comment:
Not sure how to edit this instance of \omegea on my phone, so I'll just note it here, accept, and then leave the squash and merge for afterwards.

Reply:
```suggestion
{P18} for $\omega$-covers. This property is
```